svt-av1/.SRCINFO

```diff
@@ -11,7 +11,7 @@ pkgbase = svt-av1
 	makedepends = nasm
 	makedepends = ninja
 	depends = glibc
-	source = git+https://gitlab.com/AOMediaCodec/SVT-AV1.git#tag=59645eea34e2815b627b8293aa3af254eddd0d69
+	source = git+https://github.com/gianni-rosato/svt-av1-psy.git#tag=5cac47c1eaee9c5229cdf693bf26e90ff6c54b40
```

Comment:
From checking the repo, I saw that this 5cac47c1eaee9c5229cdf693bf26e90ff6c54b40 is the commit of the RC tag. I don't know if tagging by a commit will work (it may), but it would be better to define as commit, for standard purposes.

Reply:
I will push a fix once I've fixed it.

Reply:
I pushed a fix that tags the proper commit: `svt-av1 v2.0.0.rc2.r0.g5cac47c1-1`